ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/eventstream/services/EventStreamServiceImpl.java

```diff
@@ -34,9 +40,10 @@ public List<EventStreamResponse> retrieveAllEventStreams() {
 		return eventStreamCollection.retrieveAllEventStreams().stream().map(eventStream -> {
 			List<ViewSpecification> views = viewService.getViewsByCollectionName(eventStream.getCollection());
 			ShaclShape shaclShape = shaclShapeService.retrieveShaclShape(eventStream.getCollection());
+			Optional<DcatDataset> dataset = dcatDatasetService.retrieveDataset(eventStream.getCollection());
 			return new EventStreamResponse(eventStream.getCollection(), eventStream.getTimestampPath(),
 					eventStream.getVersionOfPath(), eventStream.getMemberType(), eventStream.isDefaultViewEnabled(),
-					views, shaclShape.getModel());
+					views, shaclShape.getModel(), dataset.orElse(null));
```

Comment:
If I understand this correctly, this will include a jena model in the Response that we provide at the controller? This, together with the shape, don't look too ideal. No need to fix it immediately, but this should be on our radar to avoid